NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt

```diff
@@ -37,7 +37,6 @@ class NeuroID private constructor(
     internal var application: Application?, internal var clientKey: String
 ) {
     private var firstTime = true
-    private var endpoint = ENDPOINT_PRODUCTION
```

Comment:
Where is ENDPOINT_PRODUCTION coming from? Should we remove it where its declared or replace it with the new constant?

Reply:
Yeah will remove ENDPOINT_PRODUCTION. It has been replaced with Constants.productionEndpoint.displayName. Nice find.